### ISSUE TITLE
Cannot find Module openlayers

### ISSUE THREAD

**Issue by FallenRiteMonk** [quentin-ol/angular2-openlayers#2]
Importing the module like:
```
...
import { AngularOpenlayersModule } from 'angular2-openlayers';

@NgModule({
  imports: [
    ...,
    AngularOpenlayersModule
  ],
...
```
result in the following error message:
```
ERROR in [default] <PROJECT_PATH>/web/node_modules/angular2-openlayers/dist/components/feature.component.d.ts:2:24 
Cannot find module 'openlayers'.

ERROR in [default] <PROJECT_PATH>/node_modules/angular2-openlayers/dist/components/geometry.components.d.ts:2:21 
Cannot find module 'openlayers'.

ERROR in [default] <PROJECT_PATH>/node_modules/angular2-openlayers/dist/components/layer.components.d.ts:2:22 
Cannot find module 'openlayers'.

ERROR in [default] <PROJECT_PATH>/node_modules/angular2-openlayers/dist/components/map.component.d.ts:2:20 
Cannot find module 'openlayers'.

ERROR in [default] <PROJECT_PATH>/node_modules/angular2-openlayers/dist/components/source.components.d.ts:2:23 
Cannot find module 'openlayers'.

ERROR in [default] <PROJECT_PATH>/node_modules/angular2-openlayers/dist/components/view.component.d.ts:2:21 
Cannot find module 'openlayers'.
```

**Comment by quentin-ol** [comment]
It looks like the openlayers type definitions are missing, my bad. 
I will add them as a dependency.

In the meantime, those are available in the `@types` npm repository : 
```
npm install @types/openlayers --save 
```

**Comment by quentin-ol** [comment]
Fixed in commit e2721f8c1b5baa0775f1393de2c2b6cf53d64772.
Available as npm package 0.1.4: `npm update` should do the trick.
Closing the issue.
